introduction.md

```diff
@@ -1 +1,4 @@
-Hi, I'm elcresan. The Javascript path in CodeSchool was great!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?